app-tasks/dags/aoi-monitoring/update_aoi_projects.py

```diff
@@ -28,24 +29,31 @@
     concurrency=os.getenv('AIRFLOW_DAG_CONCURRENCY', 24)
 )
 
+# UTILS
+def start_update(project_id):
+    bash_cmd = 'java -cp /opt/raster-foundry/jars/rf-batch.jar com.azavea.rf.batch.Main update_aoi_project {0}'.format(project_id)
 
+    subprocess.call([bash_cmd], shell=True)
+    logger.info('Checking whether %s has updated scenes available', project_id)
+    is_success = wait_for_success(project_id)
```

Comment:
Since the script is kind of a black box of sorts, I think inspecting the exit code will be enough -- with the understanding if there is some type of uncaught exception, it should be marked as unsuccessful. `subprocess.call` returns the exit code so I think you can do something like this:

```python
exit_code = subprocess.call([bash_cmd], shell=True)
is_success = exit_code == 0
```